lang_tests/run.rs

```diff
@@ -1,10 +1,12 @@
 use std::{fs::read_to_string, process::Command};
-
+use tempfile::NamedTempFile;
 use lang_tester::LangTester;
 
 const COMMENT: &str = "//";
 
 fn main() {
+    let temp = NamedTempFile::new().unwrap();
```

Comment:
I think you can simplify this to `NamedTempFile::new().unwrap().into_temp_path()`.